### PR TITLE
Bump devon_rex_base to 1.3.1 - run as user

### PR DESCRIPTION
**Pull request by yykamei**
Use devon_rex_base:1.3.1 to run runners related to Python as UNIX general user.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 
 # Install pyenv
+USER $RUNNER_USER
 RUN curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-installer | bash
-
-ENV PYENV_ROOT /root/.pyenv
-ENV PATH $PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
+ENV PYENV_SHIMS_BIN="$RUNNER_USER_HOME/.pyenv/shims"
+ENV PYENV_BIN="$RUNNER_USER_HOME/.pyenv/bin"
+ENV PATH="$PYENV_SHIMS_BIN:$PYENV_BIN:$PATH"
```
Comment (ybiquitous) [inline Dockerfile]: [question] What difference between `ENV PATH $FOO` and `ENV PATH="$FOO"` formats?

Comment (yykamei) [reply Dockerfile]: According to the [document](https://docs.docker.com/engine/reference/builder/#env), the first one assumes the entire string after the first argument value, so `ENV PATH a b` means `PATH="a b"`.

Comment (ybiquitous) [reply Dockerfile]: Thanks, I see! I prefer `ENV myName="John Doe"` over `ENV myName John Doe`, too. 😏

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 # Install python 2 and 3
 RUN pyenv install 2.7.16 \
   && pyenv install 3.7.2 \
   && pyenv rehash        \
   && pyenv global 3.7.2
+
+# Edit secure_path to include PATH required by Python
+USER root
+RUN sed -i -e '/secure_path/d' /etc/sudoers && \
+    echo "Defaults secure_path=\"$GEM_HOME/bin:$PYENV_SHIMS_BIN:$PYENV_BIN:/usr/local/bin:/usr/bin:/bin\"" >> /etc/sudoers
```
Comment (ybiquitous) [inline Dockerfile]: [question] Is there a possibility that `$GEM_HOME` overwrites `$PYENV_SHIMS_BIN` or `$PYENV_BIN `?

E.g. `$GEM_HOME/foo` overwrites `$PYENV_SHIMS_BIN/foo`.

Comment (yykamei) [reply Dockerfile]: Yes, there is, of course. We should try to avoid such a situation by testing in each runner.

Comment (ybiquitous) [reply Dockerfile]: OK, I see. 👍

**Pull request merged**